From ccb008b459fb1fce6614e33c44ee7faed9a366a2 Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <me@zygoon.pl>
Date: Thu, 17 Jan 2019 15:46:00 +0200
Subject: [PATCH 5/9] advisor,errtracker: use upstream bolt package

Upstream snapd uses a fork of the bolt package that carries additional
patches for bugs that were discovered by snapd developers. Bolt itself
appears to be an abandoned project and is not accepting any new patches.

In various distributions the upstream bolt package may or may not have
been patched but the forked version was definitely not packaged. As
such, to build snapd in Debian the upstream bolt package name must be
used.

Signed-off-by: Zygmunt Krynicki <me@zygoon.pl>
---
 advisor/backend.go       | 2 +-
 errtracker/errtracker.go | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/advisor/backend.go b/advisor/backend.go
index d74aa14fe..42a4ba051 100644
--- a/advisor/backend.go
+++ b/advisor/backend.go
@@ -25,7 +25,7 @@ import (
 	"path/filepath"
 	"time"
 
-	"github.com/snapcore/bolt"
+	"github.com/boltdb/bolt"
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/osutil"
diff --git a/errtracker/errtracker.go b/errtracker/errtracker.go
index d7507a9a3..1910b7d7c 100644
--- a/errtracker/errtracker.go
+++ b/errtracker/errtracker.go
@@ -33,7 +33,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/snapcore/bolt"
+	"github.com/boltdb/bolt"
 	"gopkg.in/mgo.v2/bson"
 
 	"github.com/snapcore/snapd/arch"
-- 
2.17.1

